widget/charts/tsconfig.build.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Please use this config instead:
```
{
  // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
  "extends": "../../tsconfig.libnext.json",
  "include": ["src"],
  "compilerOptions": {
    "outDir": "dist",
    "rootDir": "./src",
    "jsx": "react",
    "skipLibCheck": true
  }
}
```

and fix errors.

Reply:
fixed